sprint-planning/readme.md

```diff
@@ -22,7 +22,7 @@ General guidance:
 
 Specific roles:
 
-- [Scrum Master](https://www.agilealliance.org/glossary/scrum-master):
+- [Process Lead](https://www.agilealliance.org/glossary/scrum-master):

```

Comment:
Does it still make sense to link out to Agile Alliance here or does that cause confusion?

Reply:
That's a great point, removing and will resolve when gone

Reply:
Pushed, and added a little more wordiness to the bullet point referencing this bit. Do you want to review and close this issue if it meets your goal?